src/main/kotlin/exercise3/task1/BalancedBrackets.kt

```diff
@@ -26,9 +27,32 @@ package exercise3.task1
 
 
 internal fun isExpressionBalanced(expression: String): Boolean {
-    TODO("Implement me!!!")
+    val chars = Stack<Char>()
```

Comment:
The solution is correct. 👏🏻 
Please now try solving it in the functional way. :)